src/analysis/retail/shaman/elemental/modules/talents/Ascendance.tsx

```diff
@@ -18,31 +19,31 @@ class Ascendance extends Analyzer {
   justEnteredAscendance = false;
   checkDelay = 0;
   numCasts = {
-    [SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id]: 0,
-    [SPELLS.LAVA_BURST.id]: 0,
+    [TALENTS.ASCENDANCE_ELEMENTAL_TALENT.id]: 0,
+    [TALENTS.LAVA_BURST_TALENT.id]: 0,
     [SPELLS.EARTH_SHOCK.id]: 0,
```

Comment:
this is a talent now, right? should remove `SPELLS.EARTH_SHOCK` and replace with `TALENTS.EARTH_SHOCK_TALENT`

Reply:
Thanks for that. For future reference, are the actuall spell Ids just the talent Ids now? So is there a time when Spells.Earth_shock would be used?